filebeat/module/iis/w3svc/ingest/pipeline.json

```diff
@@ -0,0 +1,46 @@
+{
+  "description": "Pipeline for parsing iis w3svc logs",
```

Comment:
This description refers to "iis" logs. This should read "IIS". For consistency with the Apache2 and Nginx modules, I think that this description should read:

      "Pipeline for parsing IIS access logs. Requires the geoip and user_agent plugins."